pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -913,6 +915,10 @@ async def dispense(
     for op in dispenses:
       if does_volume_tracking():
         if not op.resource.tracker.is_disabled:
+          # Update the liquid history of the tip to reflect new liquid
+          if (not op.tip.tracker.is_cross_contamination_tracking_disabled) and \
+              not op.resource.tracker.is_cross_contamination_tracking_disabled:
+            op.tip.tracker.liquid_history = op.resource.tracker.liquid_history
```

Comment:
should `op.tip.tracker.liquid_history` not be a `set` that is extended?

Reply:
It can be. In my head, I want the liquid history to be literally changed to the destination's history because it seems more permanent. I don't think the behavior of the code will change in either case though.